GVRf/Framework/src/org/gearvrf/scene_objects/GVRTextViewSceneObject.java

```diff
@@ -335,12 +369,56 @@ public int getGravity() {
         return mTextView.getGravity();
     }
 
+    /**
+     * Set the refresh frequency of this scene object.
+     * 
+     * @param frequency
+     *            The refresh frequency of this TextViewSceneObject.
+     */
+    public void setRefreshFrequency(IntervalFrequency frequency) {
+        switch (frequency) {
+        case HIGH:
+            mRefreshInterval = HIGH_REFRESH_INTERVAL;
+            break;
+        case MEDIUM:
+            mRefreshInterval = MEDIUM_REFRESH_INTERVAL;
+            break;
+        case LOW:
+            mRefreshInterval = LOW_REFRESH_INTERVAL;
+            break;
+        default:
+            break;
+        }
+    }
+
+    /**
+     * Get the refresh frequency of this scene object.
+     * 
+     * @return The refresh frequency of this TextViewSceneObject.
+     */
+
+    public IntervalFrequency getRefreshFrequency() {
+        switch (mRefreshInterval) {
+        case 10:
+            return IntervalFrequency.HIGH;
+        case 20:
+            return IntervalFrequency.MEDIUM;
+        default:
+            return IntervalFrequency.LOW;
```

Comment:
to be align with this definition of the default, mRefreshInterval = MEDIUM_REFRESH_INTERVAL;
        case MEDIUM_REFRESH_INTERVAL:
        default:
            return IntervalFrequency.MEDIUM;